espnet/nets/pytorch_backend/e2e_tts_fastspeech.py

```diff
@@ -583,6 +591,8 @@ def _forward(
         d_masks = make_pad_mask(ilens).to(xs.device)
         if is_inference:
             d_outs = self.duration_predictor.inference(hs, d_masks)  # (B, Tmax)
+            if alpha is not None:
+                d_outs = torch.round(d_outs.float() * alpha).long()
```

Comment:
I believe you implemented this a while ago. Please see https://github.com/espnet/espnet/blob/6003c7f10cda7000441d4b2c78109a524eeb0228/espnet/nets/pytorch_backend/fastspeech/length_regulator.py#L54-L55

So, how about just passing `alpha` to the length regulator?

Reply:
Ah ha. You are right...